tests/api/features/steps/lists_steps.py

```diff
@@ -0,0 +1,27 @@
+from behave import given, step
+from main.trello.api.lists_manager import ListsManager
+from main.utils.behave_helpers import fill_payload
+
+manager = ListsManager()
+
+
+@given('I created a board')
+def step_impl(context):
+    pass
+
+
```

Comment:
There already a step in master's board_steps.py with the same purpose ('A board is created')
```suggestion

```

Reply:
Fixed in new commit.